### PR TITLE
Create clau.json

### PR DESCRIPTION
**Pull request by clqu**

### CODE REVIEW

File: domains/clau.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "My Github pages portfolio",
+  "repo": "https://github.com/iClaudette/claudette.github.io/tree/main",
+  "owner": {
+    "username": "iClaudette",
+    "email": "claudette@voiddevs.org"
+  },
+  "record": {
+    "CNAME": "claudette.github.io"
```
Comment (phenax) [inline domains/clau.json:9]: This is an invalid cname. Seems like there is a mismatch between your username and the name of the repo.
```suggestion
    "CNAME": "iClaudette.github.io"
```

I'd also reccommend renaming your repo to `iClaudette.github.io` but it'll work either way.

**Review by phenax — CHANGES_REQUESTED** [review]

**Comment by phenax** [comment]
You have 2 pr's open for the same subdomain. Can you keep the one you want and close the other? #622 and #632

**Comment by Codeize** [comment]
Hi @iClaudette . Are you still wanting to continue with this subdomain? Thanks

**Comment by TheBotlyNoob** [comment]
Closing because of inactivity

**Pull request closed without merging**